pytest_deep_analysis/runtime/collectors.py

```
@@ -0,0 +1,194 @@
+    tracking which mock methods were called during test execution.
+    """
+
+    MOCK_VERIFY_METHODS = {
+        "assert_called",
+        "assert_called_once",
+        "assert_called_with",
+        "assert_called_once_with",
+        "assert_any_call",
+        "assert_has_calls",
+        "assert_not_called"
+    }
```

Ruff (0.14.4)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)